openupgrade_scripts/scripts/crm/18.0.1.8/post-migrate.py

```diff
@@ -0,0 +1,11 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.load_data(env, "crm", "18.0.1.8/noupdate_changes.xml")
+    openupgrade.delete_record_translations(
```

Comment:
don't use this, only used for email templates or records that change their translation fields